lib/fluent/plugin_id.rb

```diff
@@ -47,11 +47,15 @@ def plugin_id_for_test?
     end
 
     def plugin_id_configured?
-      @_id_configured
+      if instance_variable_defined?("@_id_configured")
+        @_id_configured
+      end
     end
 
     def plugin_id
-      @id ? @id : "object:#{object_id.to_s(16)}"
+      if instance_variable_defined?("@id")
```

Comment:
This method should return `"object:#{object_id.to_s(16)}"` if instance_variable `@id` is not defined.